ranger-core/src/main/java/io/appform/ranger/core/finder/serviceregistry/ServiceRegistryUpdater.java

```diff
@@ -74,7 +73,7 @@ public void start() {
             RetryerBuilder.<Boolean>newBuilder()
                     .retryIfResult(r -> null == r || !r)
                     .build()
-                    .call(initialRefreshCompleted::get);
+                    .call(() -> serviceRegistry.getInitialRefreshCompleted().get());
```

Comment:
This will need changed based on another comment on ServiceRegistry

Reply:
Got changed with the other change